mappings/net/minecraft/world/chunk/WorldChunk.mapping

```diff
@@ -1,7 +1,5 @@
 CLASS bsz net/minecraft/world/chunk/WorldChunk
-	CLASS bsz$a AccessType
-		FIELD a CREATE Lbsz$a;
-		FIELD c GET Lbsz$a;
+	CLASS bsz$a CreationType
```

Comment:
When this is used, is a chunk always created? If it's still `GET` and `CREATE`, `AccessType` seems to make more sense

Reply:
The official Mojang names are IMMEDIATE, QUEUED and CHECK.